service/src/test/java/org/cbioportal/service/impl/MutationServiceImplTest.java

```diff
@@ -47,16 +47,17 @@ public void getMutationsInMolecularProfileBySampleListId() throws Exception {
         Mockito.when(mutationRepository.getMutationsInMolecularProfileBySampleListId(MOLECULAR_PROFILE_ID, 
             SAMPLE_LIST_ID, Arrays.asList(ENTREZ_GENE_ID_1), null, PROJECTION, PAGE_SIZE, PAGE_NUMBER, SORT, DIRECTION))
             .thenReturn(expectedMutationList);
-        Mockito.doAnswer(invocationOnMock -> {
+        //TODO: varify chromosome from reference genome gene
+        /*Mockito.doAnswer(invocationOnMock -> {
```

Comment:
since these are `TODO`s I suppose it's okay to leave them commented here but I do believe these should be addressed in this PR

Reply:
@jjgao made a comment: no need to perform the check, since is not necessary. Will remove those notes. thanks!

Reply:
removed, since now we use **chr** recorded in the mutation_event table, so we do need to set a mockito chr value.